packages/server/graphql/public/mutations/startRecurrence.ts

```diff
@@ -37,37 +40,88 @@ const startRecurrence: MutationResolvers['startRecurrence'] = async (
     return standardError(new Error('Meeting is not a team prompt meeting'), {userId: viewerId})
   }
 
-  if (meeting.meetingSeriesId) {
-    return standardError(new Error('Meeting already has meeting series'), {userId: viewerId})
-  }
-
   // Next meeting start is tomorrow at 9a UTC
   // :TODO: (jmtaber129): Determine this from meeting series configuration.
-  const startDate = new Date(
+  const nextMeetingStartDate = new Date(
     Date.UTC(now.getUTCFullYear(), now.getUTCMonth(), now.getUTCDate() + 1, 9)
   )
   const recurrenceRule = new RRule({
     freq: RRule.WEEKLY,
     byweekday: [RRule.MO, RRule.TU, RRule.WE, RRule.TH, RRule.FR],
-    dtstart: startDate
+    dtstart: nextMeetingStartDate
   })
```

Comment:
+1 Assuming we change the restart logic to not use `nextMeetingStartDate`, this will only be used when initializing a meeting series, so let's move this down below the `if` block